aries_cloudagent/config/default_context.py

```diff
@@ -133,4 +133,5 @@ async def load_plugins(self, context: InjectionContext):
             plugin_registry.register_plugin(plugin_path)
 
         # Register message protocols
-        await plugin_registry.init_context(context)
+        plugins_config = self.settings.get("plugins_config", {})
+        await plugin_registry.init_context(context, plugins_config)
```

Comment:
The settings should be available in the context already, shouldn't they? I don't think it's necessary to alter the `init_context` method signature